src/uk/gov/defra/ffc/Utils.groovy

```diff
@@ -114,4 +114,14 @@ class Utils implements Serializable {
       ctx.sh(returnStatus: true, script: script)      
     }
   }
+
+  static def checkCredentialsExist(ctx, id) {
```

Comment:
Suggest changing the `def` to `Boolean`.  I know in a lot of the existing classes we're just using `def` for everything but ideally I'd like to transition those to proper datatypes over time.  And if we add new stuff correctly defined then that task will be less effort.

So I'd change this line to `static Boolean checkCredentialsExist(def ctx, String id)`

Reply:
Agreed.